src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift

```diff
@@ -124,14 +124,16 @@ class DynamicTableViewImageCardCell: UITableViewCell {
 		cellImage.heightAnchor.constraint(equalToConstant: 130).isActive = true
 	}
```

Comment:
I think that file might be gone. Changed need to be applied to `ExposureSubmissionImageCardCell`.